pagefaultgames/pokerogue
# Balance endless tokens

## What are the changes?

- Removed Sleep and Freeze tokens
- Rebalanced the others to be slightly less overbearing

## Why am I doing these changes?



Sleep and freeze tokens are widely agreed to make endless unfun. It’s honestly been a long time coming but the recent berry rebalancing made it much more urgent to finally rebalance these tokens.
These changes were made in accordance to discussions with the balance team and other devs. 
[This is where this change was initiated.](https://discord.com/channels/1125469663833370665/1229525892624154684/1246871204149858384)

## What did change?


- Sleep and freeze tokens have been erased from existence
- Reduced paralyze token proc chance from 10% to 2.5% and increased stack limit from 5 to 10
- For both burn and poison tokens, reduced proc chance from 10% to 5% and increased stack limit from 5 to 10
- Reduced endure token proc chance from 2.5% to 2%
- Reduced status heal token proc chance from 10% to 5%
- Reduced enemy heal token stack limit from 15 to 10
- Total max amount of tokens remains the same

### Screenshots/Videos


![image](https://github.com/pagefaultgames/pokerogue/assets/36677462/b64405ef-a631-4d56-ac79-64f7f75485dc)

## How to test the changes?




Import a current endless run and try to get another token. Try to set status on the opponent and take some hits to check the chances. You could override the opponent moves to Brutal Swing so there’s 3 proc chances in one move.

## Checklist
